libafl/src/mutators/mutations.rs

```diff
@@ -1138,7 +1138,7 @@ where
         }
 
         // We don't want to use the testcase we're already using for splicing
-        let idx = random_corpus_id!(state.corpus(), state.rand_mut());
+        let idx = random_corpus_id!(state.corpus(), state.rand_mut(), false);
```

Comment:
For Crossover/Splicing, I think we also need to include disabled (or maybe it should be configurable)

cc @vanhauser-thc ?

Reply:
Maybe even have a random chance to pick a disabled one (so a lower probability to pick them, overall)
Probably needs to be fuzzbench-ed again

Reply:
cc @tokatoka as well

Reply:
I've enabled the inclusion of disabled entries in the meantime.

Reply:
i'll check after i finish my pr